scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -52,12 +74,37 @@ public static BuildTargetResult UpdateBuild(BuildTargetContext c)
 
         // Moving PrepareTargets.RestorePackages after PackagePkgProjects because managed code depends on the
         // Microsoft.NETCore.App package that is created during PackagePkgProjects.
-        [Target(nameof(PrepareTargets.Init), nameof(CompileCoreHost), nameof(PackagePkgProjects), nameof(PrepareTargets.RestorePackages), nameof(CompileStage1), nameof(CompileStage2))]
+        [Target(nameof(PrepareTargets.Init), nameof(CompileCoreHost), nameof(PackagePkgProjects), nameof(CompileTargets.GenerateStubHostPackages), nameof(PrepareTargets.RestorePackages), nameof(CompileStage1), nameof(CompileStage2))]
         public static BuildTargetResult Compile(BuildTargetContext c)
         {
             return c.Success();
         }
 
+        [Target]
+        public static BuildTargetResult GenerateStubHostPackages(BuildTargetContext c)
```

Comment:
It isn't really a naming issue, it is a "why" issue.  Someone coming into this code would easily ask themselves, "why in the world are they generating a "stub host package"?". This would tell them the answer.

A link to an issue would suffice.